src/splitInteger.test.js

```diff
@@ -1,21 +1,42 @@
 'use strict';
 
-const splitInteger = require('./splitInteger');
+describe('splitInteger', () => {
+  const splitInteger = require('./splitInteger');
 
-test(`should split a number into equal parts
+  test(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
+    expect(splitInteger(6, 2))
+      .toEqual([
+        3, 3,
+      ]);
+  });
 
-});
-
-test(`should return a part equals to a value
+  test(`should return a part equals to a value
   when splitting into 1 part`, () => {
-
-});
-
-test('should sort parts ascending if they are not equal', () => {
-
-});
-
-test('should add zeros if value < numberOfParts', () => {
-
+    expect(splitInteger(8, 1))
+      .toEqual([
+        8,
+      ]);
+  });
+
+  test('should sort parts ascending if they are not equal', () => {
+    expect(splitInteger(17, 4))
+      .toEqual([
+        4, 4, 4, 5,
+      ]);
+  });
+
+  test('should sort parts ascending if they are not equal', () => {
```

Comment:
you have two identical tests

Reply:
Yep, here we have different checks but identical summaries. Please, change summary in the test